rust_src/src/frames.rs

```diff
@@ -77,4 +91,68 @@ pub fn selected_frame() -> LispObject {
     unsafe { LispObject::from(current_frame) }
 }
 
+/// Return non-nil if OBJECT is a frame which has not been deleted.
+/// Value is nil if OBJECT is not a live frame.  If object is a live
+/// frame, the return value indicates what sort of terminal device it is
+/// displayed on.  See the documentation of `framep' for possible
+/// return values.
+#[lisp_fn]
+fn frame_live_p(object: LispObject) -> LispObject {
```

Comment:
Hmmm, I think this should be clear. But I just looked at our first bug issues and wonder how we can make them more appealing.

Where would I document a list that contains frequent c macros/functions and the rust counterparts. I think it's too much for the walkthrough ? 

It would make things a lot easier for newcomers if they could lookup how to use `XSETFASTINT` for example.

Reply:
I added a porting cookbook to the wiki based on the README. I need to link to it in the README.